src/test/java/swinglib/JPanelBuilderTest.java

```diff
@@ -0,0 +1,88 @@
+package swinglib;
+
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.core.IsNull.notNullValue;
+
+import java.awt.BorderLayout;
+import java.awt.GridBagLayout;
+import java.awt.GridLayout;
+import java.awt.Insets;
+import java.util.Arrays;
+
+import javax.swing.JComponent;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.border.EmptyBorder;
+
+import org.hamcrest.core.Is;
+
+import org.junit.Test;
+
+public class JPanelBuilderTest {
+
+  @Test
+  public void minBuildCase() {
+    assertThat(JPanelBuilder.builder().build(), notNullValue());
+  }
+
+  @Test
+  public void testBorder() {
+    final int borderWidth = 3;
+    final JPanel panel = JPanelBuilder.builder()
+        .border(JPanelBuilder.BorderType.EMPTY)
+        .borderWidth(borderWidth)
+        .build();
+
+    assertThat(panel.getBorder(), instanceOf(EmptyBorder.class));
+
+    final Insets borderInsets = ((EmptyBorder) panel.getBorder()).getBorderInsets();
+    assertThat(borderInsets.top, Is.is(borderWidth));
+    assertThat(borderInsets.bottom, Is.is(borderWidth));
+    assertThat(borderInsets.left, Is.is(borderWidth));
+    assertThat(borderInsets.right, Is.is(borderWidth));
```

Comment:
Nit: These tests would read more easily if `org.hamcrest.core.Is.is` or `org.hamcrest.Matchers.is` were statically imported.